.gitignore

```diff
@@ -18,6 +18,9 @@ vendor/
 # Build binaries
 bin/
 
+# Used to serve swagger documentation
+!swagger/**/*.html
```

Comment:
Hm, does this has to be shipped around with the binary out of interest?  i.e. user needs binary, config file and swagger directory?

Reply:
Are you suggesting the docs be hosted some place instead of users self-hosting it? I think that's definitely possible as well. I'm not sure how useful having an offline spec would be for users.